js/main.js

```diff
@@ -0,0 +1,65 @@
+$(function() {
```

Comment:
Technically, this `jQuery.ready` pattern isn't necessary since you are including you scripts at the bottom of the page. Nothing wrong with it, but it isn't required.